ecosystem/sep-0006.md

```diff
@@ -67,6 +67,50 @@ Valid CORS headers are necessary to allow web clients from other sites to use th
 Access-Control-Allow-Origin: *
 ```
 
+## Recommendations
+
+SEP-6 lays out many options for how deposit and withdrawal can work. These are recommendations for getting a wallet or anchor implementation working with minimal effort while providing a great user experience.
+
+### Basic Wallet Implementation
+
+* Identify assets you want to support manually in advance
```

Comment:
This is interesting. I think that if a wallet chooses to whitelist assets, they should do that in the balance/trustline level. If I have a trustline to asset X, I expect the wallet to present the deposit/withdraw flow that X issuer provides.

Some wallets might choose to abstract away the existence of trustlines or support just one currency, that changes things a bit but I don't think we should optimize for this use case. 

For simplicity, I would remove this point and add a qualifier in the next point.

Reply:
This is more about only whitelisting integrations that the wallet has tested and confirmed work. I'll make it more clear.